app/javascript/utils/attribute-types.js

```diff
@@ -0,0 +1,9 @@
+const attributeTypes = {
+  'integer': { 'name': 'Número', 'type': 'number' },
+  'string': { 'name': 'Texto', 'type': 'text' },
+  'date': { 'name': 'Fecha', 'type': 'date' },
+  'select': { 'name': 'Tag', 'type': 'select' },
+  'boolean': { 'name': 'Booleano', 'type': 'boolean' },
+};
+
+export default attributeTypes;
```

Comment:
Buen idea la extracción a una util 💯    lo que si, en Javascript no es necesario poner las llaves de los objetos entre comillas si es que no contienen caracteres inválidos en nombramiento de variables:

```suggestion
const attributeTypes = {
  integer: { name: 'Número', type: 'number' },
  string: { name: 'Texto', type: 'text' },
  date: { name: 'Fecha', type: 'date' },
  select: { name: 'Tag', type: 'select' },
  boolean: { name: 'Booleano', type: 'boolean' },
};

export default attributeTypes;
```